WordPress/UITestsFoundation/BaseScreen.swift

```diff
@@ -69,24 +69,8 @@ extension BaseScreen {
         }
     }
 
-    /// Scroll an element into view within another element.
-    /// scrollView can be a UIScrollView, or anything that subclasses it like UITableView
-    ///
-    /// TODO: The implementation of this could use work:
-    /// - What happens if the element is above the current scroll view position?
-    /// - What happens if it's a really long scroll view?
     public func scrollElementIntoView(element: XCUIElement, within scrollView: XCUIElement, threshold: Int = 1000) {
-
-        var iteration = 0
-
-        while !element.isFullyVisibleOnScreen && iteration < threshold {
-            scrollView.scroll(byDeltaX: 0, deltaY: 100)
-            iteration += 1
-        }
-
-        if !element.isFullyVisibleOnScreen {
-            XCTFail("Unable to scroll element into view")
-        }
+        element.scrollIntoView(within: scrollView, threshold: threshold)
```

Comment:
You might notice that this code is part of `BaseScreen`, which this train of work aims to remove.

This change here moved the logic to a dedicated extension (https://github.com/wordpress-mobile/WordPress-iOS/pull/17360/commits/feb932e32024b5933d8ec39a3b412ad27eedda46) so that both `BaseScreen` and `ScreenObject` could share it in the context of this project.

Better than copy-pasting, in particular because this code is fit to live in the XCUITestHelpers project.